include/mull/Mutators/NegateConditionMutator.h

```diff
@@ -45,5 +45,6 @@ class NegateConditionMutator : public Mutator {
 
 private:
   std::vector<std::unique_ptr<irm::IRMutation>> lowLevelMutators;
+  irm::NegateXORReplacement negateXorMutator;
```

Comment:
This looks a bit unusual since NegateXOR is rather different compared to the `lowLevelMutators`.

Does it mean that we have to introduce another high-level Mutator?